From: Werner Lemberg <wl@gnu.org>
Date: Thu, 5 Dec 2013 14:01:48 +0000 (+0100)
Subject: Issue 3694: Use standard inclusion scheme for FreeType headers.
X-Git-Url: http://git.savannah.gnu.org/gitweb/?p=lilypond.git;a=commitdiff_plain;h=7705e46966bfa05015fb9fb20c68da844ab88028;hp=88435714e762229ebe2715bcf6863eb89c1f8dff

Issue 3694: Use standard inclusion scheme for FreeType headers.

The most recent FreeType release (2.5.1) has changed locations for header
files.  Using the standard way, this is not visible to applications.
---

diff --git a/lily/freetype-error.cc b/lily/freetype-error.cc
index 00e5cae..88af76c 100644
--- a/lily/freetype-error.cc
+++ b/lily/freetype-error.cc
@@ -31,7 +31,7 @@ const struct Freetype_error_message
   const char *err_msg;
 } ft_errors[] =
 
-#include <freetype/fterrors.h>
+#include FT_ERRORS_H
 
   ;
 
diff --git a/lily/open-type-font.cc b/lily/open-type-font.cc
index df6a744..837a1f2 100644
--- a/lily/open-type-font.cc
+++ b/lily/open-type-font.cc
@@ -23,7 +23,7 @@
 
 using namespace std;
 
-#include <freetype/tttables.h>
+#include FT_TRUETYPE_TABLES_H
 
 #include "dimensions.hh"
 #include "freetype.hh"
diff --git a/lily/pango-font.cc b/lily/pango-font.cc
index ee986fc..b148a5b 100644
--- a/lily/pango-font.cc
+++ b/lily/pango-font.cc
@@ -22,7 +22,7 @@
 #define PANGO_ENABLE_BACKEND
 
 #include <pango/pangoft2.h>
-#include <freetype/ftxf86.h>
+#include FT_XFREE86_H
 
 #include <map>
 #include <cstdio>
diff --git a/lily/ttf.cc b/lily/ttf.cc
index eaeb67a..6d7f97b 100644
--- a/lily/ttf.cc
+++ b/lily/ttf.cc
@@ -20,7 +20,7 @@
 #include <cstdio>
 #include "freetype.hh"
 
-#include <freetype/tttables.h>
+#include FT_TRUETYPE_TABLES_H
 
 #include "international.hh"
 #include "memory-stream.hh"
